src/core/View.js

```diff
@@ -467,15 +469,17 @@ export default class View extends Component {
       }
     }
 
-    if (this.cubeAxes.setVisibility(showCubeAxes)) {
-      Array.from(this.cubeAxes.getActors()).forEach(({ setVisibility }) =>
-        setVisibility(showCubeAxes)
-      );
-      this.renderView();
-    }
+    if (this.props.showCubeAxes) {
+      if (this.cubeAxes.setVisibility(showCubeAxes)) {
```

Comment:
Maybe do a deferred get call for the cube axes here, e.g. `if (this.getCubeAxes()...)`, where getCubeAxes will create the actor if `this.cubeAxes` is not set. That way we can support toggling showCubeAxes from a starting false state.

Reply:
Can you take a look at my latest changes?